vmngclient/api/mtt_aaa_api.py

```diff
@@ -0,0 +1,187 @@
+import logging
+
+from vmngclient.dataclasses import TenantAAA, TenantRadiusServer, TenantTacacsServer
+from vmngclient.session import vManageSession
+from vmngclient.utils.creation_tools import asdict, create_dataclass
+
+logger = logging.getLogger(__name__)
+
+
+class AAAConfigNotPresent(Exception):
+    pass
+
+
+class TenantAaaAPI:
+    """
+    Used to configure  mtt tenant management users remote servers
+    """
+
+    def __init__(self, session: vManageSession) -> None:
+        self.session = session
+
+    def __str__(self) -> str:
+        return str(self.session)
+
+    def add_aaa(self, tenant_aaa: TenantAAA) -> bool:
+        """ "
+        TenantAAA:
+            "authOrder": [ "local", "radius", "tacacs"],
+            "authFallback": true,
+            "adminAuthOrder": false,
+            "auditDisable": false,
+            "accounting": false,
+            "radiusServers": "server1"
+
+        returns bool depending on the api post call
+        """
+        url_path = "/dataservice/admin/aaa"
+        data = asdict(tenant_aaa)  # type: ignore
+
+        response = self.session.post(url=url_path, json=data)
+        logger.info(response)
```

Comment:
Not sure if this is needed. Will log something like "<Response [200]>"
vManageSession logs each response (with details) on debug level.